Foundation/NSAffineTransform.swift

```diff
@@ -9,7 +9,7 @@
 
 #if os(OSX) || os(iOS)
     import Darwin
-#elseif os(Linux)
+#elseif os(Linux) || os(Android)
```

Comment:
I [started this discussion](https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20160307/012085.html) on the swift-evolution mailing list, but was too lazy to push it forward. @compnerd is now [leading the charge](https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20160704/023534.html). 👍